File: nahomy-c2-backend/src/domain/value-objects/NivelAlerta.ts
export enum NivelAlerta {
  BAJO = 'Bajo',
  MEDIO = 'Medio',
  ALTO = 'Alto',
  CRITICO = 'Critico',
}

export class NivelAlertaValue {
  constructor(private readonly value: NivelAlerta) {}

  getValue(): NivelAlerta {
    return this.value
  }
}

File: packages/server/src/domain/value_objects/nivel_alerta.py
from enum import Enum


class NivelAlerta(str, Enum):
    BAJO = "Bajo"
    MEDIO = "Medio"
    ALTO = "Alto"
    CRITICO = "Critico"


class NivelAlertaValue:
    def __init__(self, value: NivelAlerta):
        self._value = value

    @property
    def value(self) -> NivelAlerta:
        return self._value
